src/docfx/Errors.cs

```diff
@@ -706,5 +705,44 @@ public static Error DisallowedHtml(SourceInfo? source, string tag)
             public static Error DisallowedHtml(SourceInfo? source, string tag, string attribute)
                 => new Error(ErrorLevel.Info, "disallowed-html", $"HTML attribute '{attribute}' on tag '{tag}' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.", source, name: $"{tag}_{attribute}");
         }
+
+        public static class CRR
```

Comment:
consider expand the acronym:
```suggestion
        public static class DependencyRepository
```